src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -764,6 +815,13 @@ Currently, in order to export a File PFB, \`enableLimitedFilePFBExport\` must be
       return this.isPFBRunning()
         && this.state.exportingToSevenBridges;
     }
+    if (buttonConfig.type === 'export-pfb-to-url') {
+      return this.isPFBRunning()
+        && this.state.exportingPFBToURL
+        // because we can have multiple `export-pfb-to-url` buttons, only make the
+        // one the user clicked have the pending state.
+        && this.state.targetURLTemplate === buttonConfig.targetURLTemplate;
```

Comment:
should we allow user to click on another `export-pfb-to-url` button while there is another one running? 😕